src/explorer/makeCodebook.js

```diff
@@ -1,12 +1,36 @@
 import { csv as d3csv } from 'd3';
+import { merge as d3merge } from 'd3';
 
-export function makeCodebook(meta) {
-  this.codebookWrap.selectAll('*').remove();
-  var codebook = webcodebook.createCodebook(
+export function makeCodebook(explorer) {
+  explorer.codebookWrap.selectAll('*').remove();
+
+  //add the Files section to the nav for each config
+  this.current.settings.tabs = this.current.settings.tabs
+    ? d3merge([['files'], this.current.settings.tabs])
+    : ['files', 'codebook', 'listing', 'settings'];
+
+  //set the default tab to the codebook or listing view assuming they are visible
+  if (this.current.event == 'click') {
+    this.current.settings.defaultTab = this.current.settings.tabs.indexOf(
```

Comment:
This is intentional, but maybe not ideal. Clicking the file listing automatically loads the codebook - guess we could make that an option at some point, but i think the only reasonable options are the codebook and the data listing.